google/cloud/pubsub/internal/tracing_message_batch_test.cc

```diff
@@ -147,6 +147,29 @@ TEST(TracingMessageBatch, Flush) {
                                  "messaging.pubsub.message.link", 0)))))));
 }
 
+TEST(TracingMessageBatch, PublishSpanHasThreadIdAttribute) {
```

Comment:
There is something funny about verifying that the "publish" span has the `sc::kCodeFunction` attribute in every test, but having a dedicated test for the `sc::kThreadId` attribute.

Reply:
Consider combining these into one test: `PublishSpanAttributes`.